### PR TITLE
optional 3rd column in csv upload toindicate no notification, only se…

### PR DESCRIPTION
**Pull request by jlopker**
…nds notification if the amount added is greater than 0

### CODE REVIEW

**Comment by jessykate** [comment]
oh, right-- and the tests are failing. i guess they need to be updated to handle the extra column!

File: app/controllers/allocations_controller.rb [hunk 1]
```diff
@@ -27,11 +27,15 @@ def create
     group = Group.find(allocation_params[:group_id])
     user = User.find(allocation_params[:user_id])
     amount = allocation_params[:amount]
+    notify = allocation_params[:notify]
+    params[:allocation].delete(:notify)
```
Comment (jlopker) [inline app/controllers/allocations_controller.rb]: the notify data is getting removed bc we don't want to save it to the db. if i don't remove it here, it errors out when i try to save `allocation` bc there is no `notify` column on the table.

File: app/controllers/allocations_controller.rb [hunk 2]
```diff
@@ -27,11 +27,15 @@ def create
...
     render status: 403, nothing: true and return unless current_user.is_admin_for?(group)
 
     allocation = Allocation.new(allocation_params)
     if allocation.save
-      UserMailer.notify_member_that_they_received_allocation(admin: current_user, member: user, group: group, amount: amount).deliver_now
+      if notify && (amount > 0)
```
Comment (jlopker) [inline app/controllers/allocations_controller.rb]: yea i added that `if` to check for the amount being greater than zero.

File: app/controllers/allocations_controller.rb (hunk 1 above)
Comment (jessykate) [inline app/controllers/allocations_controller.rb]: ahhhh that makes sense. thanks!

**Comment by jlopker** [comment]
@jessykate ok this should be good to go!

**Pull request merged**